lib/active_merchant/billing/gateways/worldpay_online_payments.rb

```diff
@@ -107,9 +107,9 @@ def create_post_for_auth_or_purchase(token, money, options)
           'state'=>options[:billing_address]&&options[:billing_address][:state] ? options[:billing_address][:state] : '',
           'countryCode'=>options[:billing_address]&&options[:billing_address][:country] ? options[:billing_address][:country] : ''
           },
-          'customerOrderCode' => options[:order_id],
-          'orderType' => 'ECOM',
-          'authorizeOnly' => options[:authorizeOnly] ? true : false
+        'customerOrderCode' => options[:order_id],
+        'orderType' => 'ECOM',
+        'authorizeOnly' => options[:authorizeOnly] ? true : false
```

Comment:
The indentation pattern going on here seems weird/wrong; seems like 109 should be indented less, or these lines more, or something. There's also an awkward/insufficient amount of indentation here v. the closing brace on 113. Do you know what's going on here?

Reply:
These lines are aligned with line 101

Reply:
@nfarve It looks like the closing brackets on 109 and 113 need to go left two spaces so that they align with the beginning characters of their block or section. That would follow the pattern, e.g. lines 121 & 127 and 202 & 206.

Reply:
Yeah, @deedeelavinder 's got it. In addition, the alignment issues here are probably cause because the hash is starting out underindented (which isn't new with your patch); line 95 and following should really be indented two spaces further, I suspect, and the rest of this is an alignment fight as the editor's trying to make everything work out.